src/librustdoc/html/render/mod.rs

```diff
@@ -561,10 +561,21 @@ fn document_short(
     }
 }
 
-fn document_full(w: &mut Buffer, item: &clean::Item, cx: &Context<'_>) {
+fn document_full(w: &mut Buffer, item: &clean::Item, cx: &Context<'_>, is_collapsible: bool) {
     if let Some(s) = cx.shared.maybe_collapsed_doc_value(item) {
```

Comment:
This is the first time I've noticed the `maybe_collapsed_doc_value()` method.  When I dig into it, I see:

context.rs:
```rust
    /// Based on whether the `collapse-docs` pass was run, return either the `doc_value` or the
    /// `collapsed_doc_value` of the given item.
    crate fn maybe_collapsed_doc_value<'a>(&self, item: &'a clean::Item) -> Option<String> {
        if self.collapsed { item.collapsed_doc_value() } else { item.doc_value() }
    }
```

core.rs:
```rust
            if name == "collapse-docs" {
                warning.note("the `collapse-docs` pass was removed in #80261 <https://github.com/rust-lang/rust/pull/80261>");
            }
```

So it seems like the collapse-docs pass was removed, but we still have some cleanup to do here, like making this function (`document_full`) unconditionally render `item.doc_value()`. Is that accurate?

Reply:
I guess so. I'll send another PR for it so it's not noise in here.

Reply:
I opened https://github.com/rust-lang/rust/issues/85120 for this one. Same, will do it tomorrow.